lib/alchemy/config_loader.rb

```diff
@@ -0,0 +1,85 @@
+module Alchemy
+  class ConfigLoader
+    attr_reader :which
+
+    def initialize(which)
+      @which = which.sub(/\.yml$/, '')
+    end
+
+    # Reads the config file named +<which>.yml+ from +config/alchemy/+ folder.
+    #
+    def load_all
+      if paths.empty?
+        raise LoadError, "Could not find #{which}.yml file! Please run `rails generate alchemy:scaffold`"
+      else
+        # OPTIMIZE: remove duplicates?
+        paths.map(&method(:load_file)).inject(&method(:merge))
```

Comment:
This line a little but too clever and hides too much logic. Please rewrite it into something more explicit and understandable. Since we merge I don't see a need for removing duplicates.